frameworks/elastic/tests/test_sanity.py

```diff
@@ -378,7 +378,7 @@ def test_plugin_install_via_proxy() -> None:
         _install_and_run_proxy(proxy_host, proxy_port)
 
         plugin_name = "analysis-ukrainian"
-        plugins = "https://s3.amazonaws.com/downloads.mesosphere.io/infinity-artifacts/elastic/analysis-ukrainian-7.3.2.zip"
+        plugins = "https://infinity-artifacts.s3-us-west-2.amazonaws.com/elastic/analysis-ukrainian-7.4.1.zip"
```

Comment:
This test seems to consistently fail across clusters. 1) did this plugin require updating? I assume previous versioned plugins would still work for a non-major upgrade? (then again it's elastic so that might not be true).

Reply:
Hi Sam,
Without updating the artifacts test were failing , so I updated and it's now failing with proxy check , proxy check function is trying to search for `s3.amazonaws.com` in artifacts URL and I have uploaded to infinity artifacts, I tested locally by updating sanity test by grepping something which is present in infinity artifacts URL and it passed successfully , so may be if I upload artifacts to same location as current which includes `s3.amazonaws.com` in artifacts URL  then test would pass.